libtiny_tui/src/tui.rs

```diff
@@ -50,6 +50,9 @@ pub(crate) struct TUI {
     /// Color scheme
     colors: Colors,
 
+    /// Max size of message lines
```

Comment:
This might be more clear:
```suggestion
    /// Max number of message lines
```

Reply:
Thanks for the feedback. I wish I knew I could click "Commit suggestion". Never used that before!

Reply:
It's neat, right?!